docs/database_schema_diagrams/source/cover_art_details.json

```diff
@@ -1,6 +1,6 @@
 {
   "tooltip": "Tables for cover art",
-  "_comment": "It shows tables used to set a cover art for releases and release groups. These two core entities are highlighted. It also shows when tables belong to the 'cover_art_archive' schema.",
+  "_comment": "It shows tables used to set a cover art for releases and release groups. The main tables for these two entity types are highlighted. It also shows when tables belong to the 'cover_art_archive' schema.",
```

Comment:
The change makes sense based on what it's there - why do we highlight the entity types and not the cover art bits though if those are supposed to be the core of this diagram?